manifest/content.xml

```diff
@@ -73,4 +73,27 @@
     <button />
   </form>
 
+  <!--
+    Tabs
+      Required childr elements: options, option, tab-content, tab
+      Each option element requires a single content:text child element
+      The number of tab elements should be the same of the number of option element
+  -->
+  <tabs>
```

Comment:
I haven't looked at the android control I would use for this, but I imagine it would be a similar situation for me. I think adding suggestions would be beneficial. I would want it to only be a suggestion, not a requirement though.